lib/libutils/ext/include/confine_array_index.h

```diff
@@ -0,0 +1,102 @@
+// Copyright 2019 The Fuchsia Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/*
+ * Content of LICENSE file mentioned above:
+Copyright 2019 The Fuchsia Authors. All rights reserved.
```

Comment:
Original LICENSE file mentions 2016 as copyright date, should it be 2016 or 2019 here?

Reply:
It's 2019 in the version linked in the commit message and this file is from 2019 too so 2019 should be accurate.